sdk/eventgrid/Azure.Messaging.EventGrid/tests/TestPayload.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.Messaging.EventGrid.Tests.SessionRecords
```

Comment:
hmm.. why the SessionRecords in the namespace?

Reply:
Oh my mistake I accidentally created it in that folder and moved it out after